templates/report.html

```diff
@@ -14,9 +14,12 @@
   <title>{% if lang == "en" %}Replay Examination{% else %}牌譜検討{% endif %}</title>
 </head>
 
-<body>
+<body {% if layout == "horizontal" %} horizontal="1" {% endif %} >
   <h1>{% if lang == "en" %}Replay Examination{% else %}牌譜検討{% endif %}</h1>
 
+  <input type="radio" name="style" id="radio_style_v" {% if layout == "vertical" %} checked="true" {% endif %} onclick="toggleStyle()" />vertical
+  <input type="radio" name="style" id="radio_style_h" {% if layout == "horizontal" %} checked="true" {% endif %} onclick="toggleStyle()" />horizontal
```

Comment:
Wrapping with a `label` tag is better, like `<label><input ...>vertical</label>`.

Reply:
done